.github/workflows/pull_request.yml

```diff
@@ -128,8 +128,18 @@ jobs:
       - name: Pytest
         env:
           NEBULA_TEST_LOGS_DIR: ${{ github.workspace }}/build
+          NEBULA_TEST_RM_DIR: false
         run: ./ci/test.sh test
         timeout-minutes: 25
+      - name: Graph Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-graphd.ERROR && exit 1
+      - name: Meta Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-metad.ERROR && exit 1
+      - name: Storage Log
+        if: ${{ failure() }}
+        run: cat /tmp/nebula-*/logs/nebula-storaged.ERROR
```

Comment:
I think use `upload-artifacts` action may be better than dump log contents to screen. When we add more and more test cases, the log files will be larger and larger and then it will be difficult to lookup expected information by scroll the screen.

```yaml
    - name: Upload logs 
      uses: actions/upload-artifact@v2
      if: ${{ failure() }}
      with:	
        name: nebula-test-logs
        path:  /tmp/nebula-*/logs/
```

Reply:
Had add.